### ISSUE TITLE
1.4.3 Contrast Minimum - Contrast between map filter tab text and tab backgrounds is not sufficient

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9728]
As a user with vision issues meaning some colours are difficult to see, contrast between text and background should be sufficient (4.5:1).

**Linked Work Item:**
#9493 

**Expected Result:**
All contrast between text and background colours should be at or above a ratio of 4.5:1. This allows users with various forms of colourblindness and other vision issues to perceive all text.

**Actual Result:**
None of the text in the map filter tabs meets the ratio between text and background colours.

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/4c33c3d4-aaa9-42f0-b315-ed5b48c1bc77)

**Comment by johnatawnclementawn** [comment]
Resolved in #9773
